edenai_apis/apis/deepgram/deepgram_api.py

```diff
@@ -206,7 +206,7 @@ def audio__text_to_speech(
         if audio_format:
             base_url += f"&container={audio_format}"
 
-        if sampling_rate or sampling_rate != 0:
+        if sampling_rate and sampling_rate != 0:
```

Comment:
sorry to be samet :( 
we can actually than just write:
```python
if sampling_rate:
```

Reply:
0 is not accepted and it will return an error (if the user send 0 than it should ignore the sampling_rate)

Reply:
Yes, with the condition `if sampling_rate` he will automatically not enter the if condition (0 is considered as False also)